packages/terra-form/docs/Select.md

```diff
@@ -9,7 +9,9 @@ import React from 'react';
 import Select from 'terra-form/lib/Select';
 
 <Select
-  choices={['moo', 'boo', 'bar']}
+  choices={[ { value: 'm', display: 'moo' },
```

Comment:
This should be updated to `options={[ ...`, as well documentation below in SelectField.md.